FHEM/14_SD_WS.pm

```diff
@@ -62,6 +63,7 @@ sub SD_WS_Initialize {
   $hash->{Match}    = '^W\d+x{0,1}#.*';
   $hash->{DefFn}    = "SD_WS_Define";
   $hash->{UndefFn}  = "SD_WS_Undef";
+  $hash->{SetFn}    = 'SD_WS_Set';
```

Comment:
Am besten gleich als Referenz und nicht als String:

```suggestion
  $hash->{SetFn}    = \@SD_WS_Set;
```

Reply:
OK, wird geändert.
Da ich aber immer gerne dazu lerne :-)
Welchen Vorteil hat das?

Reply:
Du speicherst die Adresse der sub in der Variable und nicht nur einen Text, der dann erst die Adresse suchen muss.

Dafür braucht es dann auch kein no strict um den String überhaupt für den Funktionsaufruf verwenden zu können und dein Editor hat die Chance zu erkennen, dass es sich hier um eine Referenz auf eine sub handelt und nicht nur um einen string.